benchmark/make-project/README.md

```diff
@@ -0,0 +1,7 @@
+# make-project
+
+This `make-project` folder contains different files to programmatically create a new Astro project. They are created inside the `projects` folder and are gitignored. These projects are used by benchmarks for testing.
+
+Each benchmark can specify the default project to run in it's `defaultProject` export, but it can be overriden if `--project <project-name>` is passed through the CLI.
```

Comment:
```suggestion
Each benchmark can specify the default project to run in its `defaultProject` export, but it can be overriden if `--project <project-name>` is passed through the CLI.
```